### PR TITLE
added kyc token

### PR DESCRIPTION
**Pull request by batrudinych**
2 endpoints are added:
* `kyc/token/get` returns bitfinex token that can be used to redirect user to bitfinex KYC platform
* `kyc/data/get` that returns KYC status of the user (queries bitfinex platform internally). Will be used by UI to show KYC status of the user.

### CODE REVIEW

File: lib/routes/v1/kyc/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+router.post('/data/get', getSchemaValidationMw(auth), checkAuth, asyncMw(async (req, res) => {
+  const { txObj } = req.body._validated.authTx
+  const account = txObj.actions[0].authorization[0].actor
+  const result = await getBfxToken(req, account)
+
+  throwOnGrcErr(result, 'getBfxToken', 'ERR_USER')
+
+  const cplToken = result.data.cplToken
+  if (!cplToken) {
+    throw new GenericError(`grc getToken request failed: ${result.message}`)
```
Comment (robertkowalski) [inline lib/routes/v1/kyc/index.js:31]: would that crash the service?

**Comment by robertkowalski** [comment]
the build is red

**Comment by robertkowalski** [comment]
its not clear from the PR description what this PR is about. it says `added kyc token`

**Comment by batrudinych** [comment]
> the build is red

no tests are implemented yet

**Review by robertkowalski — CHANGES_REQUESTED** [review]
its not clear what the feature is going to be

**Comment by robertkowalski** [comment]
Screen Shot 2020-10-29 at 11 38 04

File: lib/routes/v1/kyc/index.js (hunk 1 above)
Comment (batrudinych) [reply lib/routes/v1/kyc/index.js:31]: it won't, errors are caught and processed:
https://github.com/bitfinexcom/moonbeam/blob/master/lib/helpers/middlewares.js#L11
https://github.com/bitfinexcom/moonbeam/blob/master/lib/helpers/middlewares.js#L263

**Comment by batrudinych** [comment]
> its not clear what the feature is going to be

added the description

**Review by robertkowalski — APPROVED** [review]
lgtm, except the travis build is red. after merging it needs some testing on staging

**Pull request merged**